SpatialGDK/Documentation/content/get-started/example-project/exampleproject-local-deployment.md

```diff
@@ -50,6 +50,9 @@ To launch a local deployment:
 3. Select **Start**. This opens a terminal window and starts a local SpatialOS deployment. Your game is ready to play when you see the output `SpatialOS ready. Access the inspector at http://localhost:21000/inspector` in your terminal window.<br/>
    ![Start]({{assetRoot}}assets/screen-grabs/toolbar/start-button.png)<br/>
    _Image: On the GDK toolbar in the Editor select **Start**_</br>
+
+    > Due to a [known issue](https://github.com/spatialos/UnrealGDK/issues/940) with the GDK, you need to follow [these steps]({{urlRoot}}/content/troubleshooting#q-my-worker-instances-are-being-disconnected-from-the-spatialos-runtime-unexpectedly-while-debugging-locally) before debugging your game using C++ or Blueprint breakpoints.
```

Comment:
Also, should we add this to the end of the porting guide?

https://docs.improbable.io/unreal/alpha/content/tutorials/tutorial-porting-guide

Reply:
@padhansell are you able to answer the questions about why this is relevant to the Example Project?

@ElleEss I like the idea of putting it in an expandable box to hide it if you're not interested in debugging (pending Patrick's answer to the question above).

Reply:
Sorry for the delay guys!

Yeah I see that this breaks the onboarding flow somewhat and makes it a bit more complicated. I personally think it's important that this is called out quite loudly, because I see it as an issue that is likely to affect people as soon as they start playing with the project (although it might just be that I happen to use breakpoints more than others). 
My main reasoning for putting it here is that it's an opportunity to make them aware of this as early as possible, and that the ability to debug your game isn't a SpatialOS feature, and so isn't something that I think people would look at our docs for. It's also pretty confusing when you hit this issue and so could waste quite a bit of time for developers.  
I think @ElleEss idea of an expanding box is probably a good approach as long as it's fairly bold and likely to be read - (perhaps something like "Are you using breakpoints?"). We could then rephrase the description to briefly explain what the problem is a bit better and then link to the solution.

What do you both think?

Reply:
After reading the next comment, I understand your point about people being unlikely to debug the Example Project now! I guess it depends a lot on what people use it for, but I would expect that people will run through it once, see it works, and then immediately try and add something to it (at which point they'd be in to the realms of debugging)

Reply:
Thanks @padhansell! Sounds like some (but not all) users reading this guide will be "into the realms of debugging" (great phrase) so it would make sense to stick this in an expandable box, with a clear heading. I'll have a go at this later today if I get the time.